IHE_ITI_Suppl_IUA.md

```diff
@@ -813,13 +815,13 @@ If the request is valid and all access policy criteria are met, the Authorizatio
 
 If the authorization request is invalid, the Authorization Server shall react as defined in [OAuth 2.1, Section 4.1.2.1].
 
-###### 3.71.4.1.3.1 Client Credential grant type
+##### 3.71.4.1.3.1 Client Credential grant type
```

Comment:
why did you change the header level to 5? The numbers indicate there are 6.

Reply:
is this to get around a limitation in markdown? I see other deep header changes too.